### DISCUSSION TITLE
Grapesjs Studio Resize Control

### DISCUSSION THREAD

**Discussion by closetgeekshow** [GrapesJS/grapesjs#5443] [Q&A] [answered]
Where does that green and orange resizer control come from? Is that an existing library? I love that thing

**Comment by artf** [comment, marked as answer]
You can render arbitrary elements on top of the canvas by using [Canvas Spots](https://grapesjs.com/docs/modules/Canvas.html#canvas-spots)

**Reply by glaprida** [reply]
Any chance you would be publishing the canvas margin and padding controls as a plugin? also, it would be nice to add the ability to rename and duplicate classes ( as in gjs studio ) to the core gjs, or at least have it as a separate plugin maybe?
